[🌲] Make local copies of LLVM headers used for RemoteInspection.

mikeash opened the pull request:
We don't want to directly include headers from the LLVM project, since they may change. Copy them locally and refer to them there instead.

rdar://112982181

mikeash committed:
[🌲] Make local copies of LLVM headers used for RemoteInspection.

We don't want to directly include headers from the LLVM project, since they may change. Copy them locally and refer to them there instead.

rdar://112982181

etcwilde reviewed (approved):
Thanks for doing this. It works as far as I can tell.

etcwilde committed:
Stdlib ArrayRef template deduction guides

Adding template deduction guides so that ArrayRef initialization works
without warnings.

mikeash committed:
Merge pull request #3 from etcwilde/ewilde/to-mash/remote-mirror-llvm-headers2

Stdlib ArrayRef template deduction guides
